modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPSuccessTest.java

```diff
@@ -392,7 +410,13 @@ public void testUpdateUserDefinedAuthenticatorOfIdP() throws JsonProcessingExcep
                 .statusCode(HttpStatus.SC_OK)
                 .body("authenticatorId", equalTo(FEDERATED_AUTHENTICATOR_ID))
                 .body("name", equalTo(new String(Base64.getDecoder().decode(FEDERATED_AUTHENTICATOR_ID))))
-                .body("definedBy", equalTo("USER"))
+                .body("isEnabled", equalTo(true))
+                .body("definedBy", equalTo(DEFINED_BY_USER))
+                .body( "tags", hasItems("Custom"))
```

Comment:
remove unnecessary space before "tags"

Reply:
This is updated with 19889d29a846f9073e796922aee6f22da20d7303